### PR TITLE
Feature/public awards page

### PR DESCRIPTION
**Pull request by prototorpedo**

### CODE REVIEW

File: app/Http/Livewire/MovieDistForm.php [hunk 1]
```diff
@@ -87,6 +87,8 @@ protected function getListeners()
         'movie.film_type' => 'required|string',
 
         'movie.imdb_url' => 'string|max:255',
+        'movie.cover_url' => 'string|max:255',
+        'movie.trailer_url' => 'string|max:255',
```
Comment (mikzalais) [inline app/Http/Livewire/MovieDistForm.php:91]: Are these two rules required here?
Because these fields don't show up on Dist form.

Comment (prototorpedo) [reply app/Http/Livewire/MovieDistForm.php:91]: They should show for editors, when editing. Is it correct?

Comment (mikzalais) [reply app/Http/Livewire/MovieDistForm.php:91]: Ok, I'll add the components. I guess we need to allow editor to edit these fields.

Comment (mikzalais) [reply app/Http/Livewire/MovieDistForm.php:91]: Ok, you already did it previously. Never mind then.

Comment (prototorpedo) [reply app/Http/Livewire/MovieDistForm.php:91]: :) I saw you splitted the rules into different buckets so I just put a check to show the fields for editor and for updates

**Pull request merged**